sklearn/metric_learning/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+The :mod:`sklearn.metric_learning` module implements metric learning models. 
+
+The algorithms that have been implemented are:
+Relevant Components Analysis (RCA)
```

Comment:
Typo! I was reading something else at the same time...thanks!
On Sep 16, 2015 4:19 AM, "B@rmaley.exe" <notifications@github.com> wrote:

> In sklearn/metric_learning/__init__.py
> <https://github.com/scikit-learn/scikit-learn/pull/5276#discussion_r39603872>
> :
>
> > @@ -0,0 +1,8 @@
> > +"""
> > +The :mod:`sklearn.metric_learning` module implements metric learning models.
> > +
> > +The algorithms that have been implemented are:
> > +Relevant Components Analysis (RCA)
>
> RCA?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/5276/files#r39603872>.
>